tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/manifest.ini

```diff
@@ -26,3 +26,5 @@ carrier = true
 [test_lockscreen_notification.py]
 
 [test_lockscreen_wake_with_notification.py]
+
+[test_lockscreen_time_check.py]
```

Comment:
Like you said on the stand up this morning, ```skip-if = device == "desktop" ``` would be needed here as we can't change the time at will on b2gdesktop.